Makefile

```diff
@@ -71,12 +71,25 @@ build-disk: build-os
 	mkdir -p $(ROOT_DIR)/build
 	$(DOCKER) run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $(ROOT_DIR)/build:/build \
 		--entrypoint /usr/bin/elemental \
-		${TOOLKIT_REPO}:${VERSION} --debug build-disk --unprivileged --expandable -n elemental-$(FLAVOR).$(ARCH) --local \
+		${TOOLKIT_REPO}:${VERSION} --debug build-disk --platform $(PLATFORM) --unprivileged --expandable -n elemental-$(FLAVOR).$(ARCH) --local \
 		--squash-no-compression -o /build ${REPO}:${VERSION}
 	dd if=$(ROOT_DIR)/build/elemental-$(FLAVOR).$(ARCH).raw of=$(ROOT_DIR)/build/elemental-$(FLAVOR).$(ARCH).img conv=notrunc
 	qemu-img convert -O qcow2 $(ROOT_DIR)/build/elemental-$(FLAVOR).$(ARCH).img $(ROOT_DIR)/build/elemental-$(FLAVOR).$(ARCH).qcow2
 	qemu-img resize $(ROOT_DIR)/build/elemental-$(FLAVOR).$(ARCH).qcow2 $(DISKSIZE) 
 
+.PHONY: build-rpi-disk
+build-rpi-disk: build-os
+ifneq ("$(PLATFORM)","linux/arm64")
+	@echo "Cannot build raspberry pi disk for $PLATFORM"
+	@exit 1
+endif
+	@echo Building ${ARCH} disk
+	mkdir -p $(ROOT_DIR)/build
+	$(DOCKER) run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $(ROOT_DIR)/examples:/examples -v $(ROOT_DIR)/build:/build \
+		--entrypoint /usr/bin/elemental \
+		${TOOLKIT_REPO}:${VERSION} --debug build-disk --platform $(PLATFORM) --cloud-init-paths /examples/tumbleweed-rpi --unprivileged --expandable -n elemental-$(FLAVOR).aarch64 --local \
```

Comment:
Please start with SLE Micro for Rancher, that's our primary target. tumbleweed is secondary

Reply:
We should probably add teal back as a flavor then, or use slem4r.. as long as we don't start publishing the artifacts again it should be fine!

Reply:
What about moving the example to leap 15.5? Leap and SLE share OS essentials and SLE and SLE Micro share them too, we should get the same exact versions of all linux-utils, core-utils, systemd, grub, dracut, etc.

Reply:
I would also vote for leap, I remember having trouble accessing the sle-micro repositories when building an early version of teal.